.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,24 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+**Description**
+<!-- A clear and concise description of what the bug is. -->
+
+**How to reproduce:**
+<!-- Steps to reproduce the behavior: -->
+
+**Expected behavior**
+<!-- A clear and concise description of what you expected to happen. -->
+
+**Environment**
+ - _OS_: 
+ - _Browser_:
+
```

Comment:
I would also add: "-Theia version"
When there is a Bug, nice to know on which version they report the issue